api/src/main/java/uk/gov/hmcts/fees/register/api/configuration/SpringSecurityConfiguration.java

```diff
@@ -37,28 +70,56 @@ public void configure(WebSecurity web) {
             "/v2/**",
             "/health",
             "/health/liveness",
-            "/info");
+            "/info","/error");
     }
 
     @Override
     @SuppressFBWarnings(value = "SPRING_CSRF_PROTECTION_DISABLED", justification = "It's safe to disable CSRF protection as application is not being hit directly from the browser")
     protected void configure(HttpSecurity http) throws Exception {
-        authCheckerFilter.setAuthenticationManager(authenticationManager());
-
         http
-            .addFilter(authCheckerFilter)
+            .addFilterAfter(userAuthVerificationFilter, BearerTokenAuthenticationFilter.class)
             .sessionManagement().sessionCreationPolicy(STATELESS).and()
             .csrf().disable()
             .formLogin().disable()
             .logout().disable()
             .authorizeRequests()
-            .antMatchers(HttpMethod.POST,"/fees-register/ranged-fees", "/fees-register/fixed-fees","/fees-register/banded-fees","/fees-register/relational-fees","/fees-register/rateable-fees", "/fees/**/versions").hasAnyAuthority("freg-editor")
+            .antMatchers(HttpMethod.POST, "/fees-register/ranged-fees", "/fees-register/fixed-fees", "/fees-register/banded-fees", "/fees-register/relational-fees", "/fees-register/rateable-fees", "/fees/**/versions").hasAuthority("freg-editor")
             .antMatchers(HttpMethod.POST, "/fees-register/bulk-fixed-fees").hasAuthority("freg-editor")
-            .antMatchers(HttpMethod.PUT, "/fees-register/ranged-fees/**", "/fees-register/fixed-fees/**","/fees-register/banded-fees/**","/fees-register/relational-fees/**","/fees-register/rateable-fees/**").hasAuthority("freg-editor")
+            .antMatchers(HttpMethod.PUT, "/fees-register/ranged-fees/**", "/fees-register/fixed-fees/**", "/fees-register/banded-fees/**", "/fees-register/relational-fees/**", "/fees-register/rateable-fees/**").hasAuthority("freg-editor")
             .antMatchers(HttpMethod.PATCH, "/fees/**/versions/**/approve").hasAuthority("freg-approver")
             .antMatchers(HttpMethod.PATCH, "/fees/**/versions/**/reject").hasAuthority("freg-approver")
             .antMatchers(HttpMethod.PATCH, "/fees/**/versions/**/submit-for-review").hasAuthority("freg-editor")
             .antMatchers(HttpMethod.DELETE, "/fees-register/fees/**", "/fees/**/versions/**").hasAnyAuthority("freg-editor", "freg-admin")
-            .antMatchers(HttpMethod.GET, "/fees-register/fees/**").permitAll();
+            .antMatchers(HttpMethod.GET, "/fees-register/fees/**").permitAll()
+            .and()
+            .oauth2ResourceServer()
+            .jwt()
+            .jwtAuthenticationConverter(jwtAuthenticationConverter)
+            .and()
+            .and()
+            /*.oauth2Client()*/;
+    }
+
+    @Bean
+    public static PropertySourcesPlaceholderConfigurer propertyPlaceHolderConfigurer() {
```

Comment:
Please check if you can remove this bean definition

Reply:
Done. The code has been removed now.